drivers/d3d12/rendering_shader_container_d3d12.cpp

```diff
@@ -94,28 +94,26 @@ uint32_t RenderingDXIL::patch_specialization_constant(
 		const uint64_t (&p_stages_bit_offsets)[D3D12_BITCODE_OFFSETS_NUM_STAGES],
 		HashMap<RenderingDeviceCommons::ShaderStage, Vector<uint8_t>> &r_stages_bytecodes,
 		bool p_is_first_patch) {
-	uint32_t patch_val = 0;
+	int64_t patch_val = 0;
```

Comment:
```suggestion
	uint64_t patch_val = 0;
```

Reply:
I had it this way at first, then Windows CI complained that I'm negating an unsigned value. I'll make it more explicit with casts.